nginx/cernopendata.conf

```diff
@@ -5,7 +5,7 @@ server {
     charset utf-8;
 
     location /static {
-        root /home/invenio/.virtualenvs/cernopendata/var/cernopendata-instance;
```

Comment:
We can still keep the venv inside docker, for the sake of harmony with bare-metal and vagrant deployments, but simply force its location to `/usr/local/var/...`